normalizer/tests/test_all.py

```
@@ -0,0 +1,147 @@
+    
+    all_passed = True
+    test_results = {}
+    
+    # 1. 住所重複削除テスト
+    print("\n[1/6] 住所重複削除テスト")
+    print("-" * 40)
+    try:
+        from test_integration import test_integration_duplicate_removal
+        test1_passed = test_integration_duplicate_removal()
+        test_results["住所重複削除テスト"] = test1_passed
+    except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)